rawJob.go

```diff
@@ -1,21 +1,21 @@
 package beanstalkworker
 
-import "strconv"
 import "time"
 import "github.com/kr/beanstalk"
 import "fmt"
 import "log"
 
-// PrioLow is the priority that jobs are released/buried with.
-const PrioLow = 1025
```

Comment:
@NevilleC you can optionally do that. By default this change will mean that the return priority is inherited from the priority of the job when it is placed on the queue. This means that this package doesn't need to know about a 'default' priority.

However some calling code may want to change the priority of the job when returning it (for example you may want to lower the priority of a failing job so that when it is re-released it doesn't jump to the head of the queue).

This will potentially require changes to the calling code, which is why I am adding it now before we roll it out to apps that need/rely on this behaviour. :)